src/content/en/tools/chrome-devtools/javascript/breakpoints.md

```diff
@@ -0,0 +1,238 @@
+  <figcaption>
+    <b>Figure 1</b>: The blue icon on top of <b>29</b> indicates that there's
+    a line-of-code breakpoint set on that line.
+  </figcaption>
+</figure>
+
+### Conditional line-of-code breakpoints {: #conditional-loc }
+
+Use a conditional line-of-code breakpoint when you know the exact region of
+code that you need to investigate, but you want to pause only when some
+other condition is true.
+
+To set a conditional line-of-code breakpoint:
+
+1. Click the **Sources** tab.
+1. Open the file containing the line of code you want to break on.
+1. Go the line of code.
+1. To the left of the line of code is the line number column. Right-click on
+   it.
+1. Select **Add conditional breakpoint**. A dialog displays underneath the
+   line of code.
+1. Enter your condition in the dialog.
+1. Press <kbd>Enter</kbd> to activate the breakpoint. An orange icon appears
+   on top of the line number column.
+
+<figure>
+  <img src="imgs/conditional-loc-breakpoint.png"
+       alt="A conditional line-of-code breakpoint."
+  <figcaption>
+    <b>Figure 2</b>: The orange icon on top of <b>32</b> indicates that there's
+    a conditional line-of-code breakpoint set on that line.
+  </figcaption>
+</figure>
+
+### Manage line-of-code breakpoints {: #manage-loc }
+
+Use the **Breakpoints** pane to disable or remove line-of-code breakpoints from
+a single location.
+
+* Check the checkbox next to an entry to disable that breakpoint.
+* Right-click over an entry to remove that breakpoint.
+* Right-click anywhere in the **Breakpoints** pane to deactivate all
+  breakpoints, disable all breakpoints, or remove all breakpoints. Disabling
+  all breakpoints is equivalent to unchecking each one. Deactivating all
+  breakpoints instructs DevTools to ignore all line-of-code breakpoints, but to
+  also maintain preserve their enabled state so that they are in the same
+  state as before when you re-activate them.
+
+<figure>
+  <img src="imgs/breakpoints-pane.png"
+       alt="The Breakpoints pane."
+  <figcaption>
+    <b>Figure 3</b>: The <b>Breakpoints</b> pane shows that there are two
+    line-of-code breakpoints: one on line 15 of <code>get-started.js</code>,
+    and another on line 32.
+  </figcaption>
+</figure>
+
+## DOM change breakpoints {: #dom }
+
+Use a DOM change breakpoint when you want to pause on the code that changes
+a DOM node or its children.
+
+To set a DOM change breakpoint:
+
+1. Click the **Elements** tab.
+1. Go the element that you want to set the breakpoint on.
+1. Right-click on the element.
+1. Hover over **Break on** then select **Subtree modifications**, **Attribute
+  modifications**, or **Node removal**.
+
+<figure>
+  <img src="imgs/dom-change-breakpoint.png"
+       alt="The context menu for creating a DOM change breakpoint."
+  <figcaption>
+    <b>Figure 4</b>: The context menu for creating a DOM change breakpoint.
+  </figcaption>
+</figure>
+
+### More information on each type of DOM change breakpoint
```

Comment:
Change to 'More about DOM change breakpoints'. Or maybe, 'Types of DOM change breakpoints'. Shorter is better for the page toc.